### ISSUE TITLE
[MSFT-249] Sample challenge page: Screen reader reads an unlabelled button next to the 'pick a file' button on the submission screen.

### ISSUE THREAD

**Issue by veshu** [importtest/ww#249]
**Steps to Reproduce**
1. Go to the Sample challenge page: https://www.topcoder.com/challenges/30095347. 2. Ensure that the screen reader is switched on. 3. Activate the 'submit' link after the registration process is completed. 4. Press the key combination 'ctrl+insert+b' to bring up a list of buttons on the page. 5. Notice that there is an 'unlabelled3' button listed next to the 'pick a file' button.

**Expected Result**
All buttons should have an appropriate label to indicate its purpose to the user.

**Actual Result**
Screen reader announces an unlabelled3 button next to the pick a file button.

**Device**: Desktop/Labtop
**Operating System**: Windows
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 1.1.1 Non-Text Content
**Screen Reader Used**: JAWS
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=088f5f5d%2D9b1c%2D4b7e%2Dacbf%2D320c257c9a9a&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FKameshwarid%2Fbug%2043
